From: Tim Gardner <tim.gardner@canonical.com>
Date: Thu, 13 Dec 2007 16:01:53 +0000 (-0700)
Subject: UBUNTU: SAUCE: hdaps module does not load on Thinkpad T61P
X-Git-Url: http://kernel.ubuntu.com/git?p=ubuntu%2Fubuntu-intrepid.git;a=commitdiff_plain;h=7b96733713e5f7d58a5cc99189c966fc603d35e9

UBUNTU: SAUCE: hdaps module does not load on Thinkpad T61P
OriginalAuthor: Klaus S. Madsen <ubuntu@hjernemadsen.org>
OriginalLocation: http://launchpadlibrarian.net/10864950/hdaps-t61.patch
Bug: 133636

Adds DMI system identifier for ThinkPad T61.

Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---

Index: 2008-devel/drivers/hwmon/hdaps.c
===================================================================
--- 2008-devel.orig/drivers/hwmon/hdaps.c
+++ 2008-devel/drivers/hwmon/hdaps.c
@@ -521,6 +521,7 @@ static struct dmi_system_id __initdata h
 	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad T42"),
 	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad T43"),
 	HDAPS_DMI_MATCH_INVERT("LENOVO", "ThinkPad T60"),
+	HDAPS_DMI_MATCH_INVERT("LENOVO", "ThinkPad T61"),
 	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad X40"),
 	HDAPS_DMI_MATCH_NORMAL("IBM", "ThinkPad X41"),
 	HDAPS_DMI_MATCH_INVERT("LENOVO", "ThinkPad X60"),
